example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+      try {
+        setError(false)
+        await props.appwrite.account.createSession(
+          email,
+          password
+        )
+        props.getUserData()
+      } catch(err) {
+        
+        setError(err.message)
+      }
+       setLoading(false)
+    }
+
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="xs"  style={{ display: props.currentPage ? "block" : "none" }}>
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+          Login
+        </Typography>
+        
+        {error && (
+             <Alert severity="error">{error}</Alert>
+      )}
+        <form className={classes.form} noValidate onSubmit={(e) => login(e)}>
+          <TextField
+          onChange={(event) => setEmail(event.target.value)} 
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            id="email"
+            label="Email Address"
+            name="email"
+            autoComplete="email"
+            autoFocus
+          />
+          <TextField
+          onChange={(event) => setPassword(event.target.value)}
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            name="password"
+            label="Password"
+            type="password"
+            id="password"
+            autoComplete="current-password"
+          />
+         
+          <Button
+            type="submit"
+            fullWidth
+            variant="contained"
+            color="primary"
+            className={classes.submit}
+            disabled={loading}
+          >
+            Login
+          </Button>
+          <Grid container>
+            
+            <Grid item>
+              <Link href="#" variant="body2" onClick={()=>props.setCurrentPage(props.currentPage)}>
+                {"Don't have an account? Sign Up"}
+              </Link>
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+    </Container>
+  );
+}
+
+export {Login}
+
+```
+
+example usage
+
+```js
+   <Login
+            appwrite={appwrite}
+            currentPage={currentPage}
+            getUserData={getUserData}
+            setCurrentPage={(currentPage) => setCurrentPage(!currentPage)}
+          />
+```
+it takes four props which is 
+
+- ```appwrite``` to connect to appwrite server using sdk
+- ```currentPage``` the state of login or sign up
+- ```setCurrentPage``` to switch between login and sign up
+- ```getUserData``` to check we already login or not
+
+here we implement login because in order for us to be able to crop an image we should login to our appwrite server and this component does exactly that.
+
+## Step 4 - Create UploadImage Component
+
+UploadImage component is used for uploading component through appwrite SDK so after we login in order for us to crop an image we should upload our image so that we can crop it later.
```

Comment:
```suggestion
UploadImage component is used for uploading component through appwrite SDK so after we login in order for us to crop an image we should upload our image so that we can crop it later.
```
```suggestion
UploadImage component is used for uploading an image using the Appwrite SDK. Uploading an image will allow us to crop it later. 
```